Online/RPCs.cs

```diff
@@ -121,49 +121,83 @@ public static void GoToDeathScreen()
                 game.GoToRedsGameOver();
                 return;
             }
-            game.GetStorySession.saveState.SessionEnded(game, false, false);
+            if (OnlineManager.lobby.isOwner) {
+                game.GetStorySession.saveState.SessionEnded(game, false, false);
```

Comment:
Add a todo to investigate client side savestate syncing

Reply:
Might not need this, didn't see the save to disk stuff when I made the comment

Reply:
having clients save to memory still doesn't fix stomach item desync so i think we have to do more